cookbooks/tian_parameterization_kinematic_slab/coupled_two_phase_tian_parameterization_kinematic_slab.prm

```diff
@@ -0,0 +1,234 @@
+# We use the tian approximation method for the solid-fluid reactions, and place a maximum
+# wt% water on the 4 lithologies.
+subsection Material model
+  set Model name                                        = reactive fluid transport
+  set Material averaging                                = harmonic average only viscosity
+
+  subsection Reactive Fluid Transport Model
+    set Base model                                       = visco plastic
+    set Reference fluid density                          = 1000 # density of water
+    set Shear to bulk viscosity ratio                    = 0.1
+    set Reference fluid viscosity                        = 1
+    set Reference permeability                           = 1e-6
+    set Exponential fluid weakening factor               = 30
+    set Fluid compressibility                            = 0
+    set Fluid reaction time scale for operator splitting = 5e4
+    set Fluid-solid reaction scheme                      = tian approximation
+    set Maximum weight percent water in peridotite       = 2
+    set Maximum weight percent water in gabbro           = 1
+    set Maximum weight percent water in MORB             = 2
+    set Maximum weight percent water in sediment         = 3
+  end
+
+  subsection Visco Plastic
+    # Use the composite flow law to simulate dislocation and diffusion creep for olivine
+    # with the parameters from Hirth & Kohlstaedt 2004.
+    set Viscosity averaging scheme                      = harmonic
+    set Viscous flow law                                = composite
+
+    set Prefactors for diffusion creep                  = 4.5e-15
+    set Stress exponents for diffusion creep            = 1.0
+    set Activation energies for diffusion creep         = 375e3
+    set Activation volumes for diffusion creep          = 8.2e-6
+
+    set Prefactors for dislocation creep                = 7.4e-15
+    set Stress exponents for dislocation creep          = 3.5
+    set Activation energies for dislocation creep       = 530e3
+    set Activation volumes for dislocation creep        = 14e-6
+
+    set Angles of internal friction                     = 0
+    set Cohesions                                       = 1e50
+    set Minimum viscosity                               = 1e19
+    set Maximum viscosity                               = 1e23
+  end
+end
+
+# Refine the mesh globally to
+subsection Mesh refinement
+  set Coarsening fraction                      = 0.05
+  set Refinement fraction                      = 0.8
+  set Initial adaptive refinement              = 1
+  set Initial global refinement                = 4
+  set Strategy                                 = composition threshold, minimum refinement function
+  set Time steps between mesh refinement       = 2
+
+  # minimum of 3 global refinements
+  subsection Minimum refinement function
+    set Function expression = 4
+  end
+
+  # refine where the porosity is bigger than 1e-6
```

Comment:
Add a note about what the values of 10 are trying to achieve (i.e., this number should not be reached so don't refine). I would actually set the value to something like 100.